internal/providers/anthropic_test.go

```
@@ -0,0 +1,92 @@
+		JSONSchema: []byte(`{"type":"object"}`),
+	})
+	require.NoError(t, err)
+	assert.Equal(t, "claude-sonnet-4-6", resp.Model)
+	assert.Equal(t, 10, resp.InputTokens)
+	assert.Equal(t, 7, resp.OutputTokens)
+	assert.JSONEq(t, `{"verdict":"pass","findings":[],"next_action":"ship"}`, string(resp.RawJSON))
+}
+
+func TestAnthropic_Review_HTTPError(t *testing.T) {
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		http.Error(w, `{"error":{"message":"rate limited"}}`, 429)
```

golangci-lint (2.12.1)

[error] ST1013: should use constant http.StatusTooManyRequests instead of numeric literal 429

(staticcheck)